src/components/v5/common/ActionSidebar/partials/forms/ArbitraryTxsForm/partials/ArbitraryTransactionsTable/hooks.tsx

```diff
@@ -0,0 +1,90 @@
+import { createColumnHelper, type ColumnDef } from '@tanstack/react-table';
+import { isAddress } from 'ethers/lib/utils';
+import React, { useMemo } from 'react';
+
+import { useMobile } from '~hooks/index.ts';
+import getMaskedAddress from '~shared/MaskedAddress/getMaskedAddress.ts';
+import { formatText } from '~utils/intl.ts';
+import { type AddTransactionTableModel } from '~v5/common/ActionSidebar/partials/forms/ArbitraryTxsForm/types.ts';
+import UserAvatar from '~v5/shared/UserAvatar/UserAvatar.tsx';
+
+import CellDescription from './CellDescription.tsx';
+
+export const useArbitraryTxsTableColumns = (): ColumnDef<
+  AddTransactionTableModel,
+  string
+>[] => {
+  const columnHelper = useMemo(
+    () => createColumnHelper<AddTransactionTableModel>(),
+    [],
+  );
+  const isMobile = useMobile();
+
+  const columns: ColumnDef<AddTransactionTableModel, string>[] = useMemo(
+    () => [
+      columnHelper.accessor('contractAddress', {
+        enableSorting: false,
+        header: () => (
+          <span className="text-sm text-gray-600">
+            {formatText({ id: 'table.row.contract' })}
+          </span>
+        ),
+        cellContentWrapperClassName: '!justify-start',
+
+        cell: ({ getValue }) => {
+          const address = getValue();
+          const isAddressValid = isAddress(address);
+          const maskedAddress = getMaskedAddress({
+            address,
+          });
+          return (
+            <span className="flex max-w-full items-center gap-2">
+              {isAddressValid && <UserAvatar userAddress={address} size={20} />}
+
+              <span className="truncate text-md font-medium text-gray-900">
+                {maskedAddress.result}
+              </span>
+            </span>
+          );
+        },
+        size: isMobile ? 100 : 30,
+      }),
+      columnHelper.display({
+        id: 'description',
+        header: () => (
+          <span className="text-sm text-gray-600">
+            {formatText({ id: 'table.row.details' })}
+          </span>
+        ),
+        cell: ({ row: { original } }) => {
+          const addressTo = getMaskedAddress({
+            address: original.to,
+            isFull: !isMobile,
+          });
+          return (
+            <CellDescription
```

Comment:
What if we make `CellDescription` just this:
![image](https://github.com/user-attachments/assets/62c2c090-40fb-4b66-94cc-84f5174c7f93)

and just render 3 `CellDescription` components a la
```JSX
<div className="w-full">
   <CellDescription title="Method" value={original.method} />
   <CellDescription title="_to (address)" value={addresTo.result} />
   <CellDescription title="_amount (uint256)" value={original.amount} /> 
</div>
```
I think it's about the same, just more JSXish :D

Reply:
I will handle this one in the scope of https://github.com/JoinColony/colonyCDapp/pull/3694 , thanks 🙌